src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VurderingRepository.kt

```diff
@@ -31,6 +31,19 @@ class VurderingRepository(private val database: DatabaseInterface) : IVurderingR
             }
         }
 
+    override fun getVurderingerBulk(
+        personidenter: List<PersonIdent>,
+    ): Map<PersonIdent, Vurdering> =
+        database.connection.use { connection ->
+            connection.prepareStatement(GET_VURDERINGER).use {
+                it.setString(1, personidenter.map { it.value }.joinToString(","))
+                it.executeQuery().toList { toPVurdering() }
+            }.associate {
+                // Den nyeste vurderingen blir valgt her siden lista er sortert med den nyeste til slutt
```

Comment:
Skjønner ikke helt hva denne kommentaren betyr 🤔

Reply:
Det kan være en eller flere vurderinger for en gitt arbeidstaker, men det er bare en av dem som blir med i resultatet her. Og det er den nyeste siden query'et bruker ORDER BY created_at ASC, og associate virker sånn at det den siste forekomsten i lista "vinner".

Reply:
Aha💡 Kan det skje at man henter ut "inaktive" vurderinger her også? Bør disse filtreres ut?

Reply:
Jeg tenker at det er noe som syfooversiktsrv må håndtere (dvs hvis det er forventet der at det skal komme tilbake en aktiv vurdering (dvs et FORHANDSVARSEL), men så kommer det noe annet tilbake - da er det syfooversiktsrv som ikke burde hatt den aktuelle personen i oversikten).